src/main/java/app/daos/MovieDAO.java

```diff
@@ -151,7 +150,7 @@ public MovieDTO update(MovieDTO movieDTO) {
                 foundMovie.setVoteAverage(movie.getVoteAverage());
             }
             if (!movie.getGenres().isEmpty()) {
-                foundMovie.setGenres(getMovieGenres(movieDTO));
+                foundMovie.setGenres(new HashSet<>(getMovieGenres(movieDTO)));
```

Comment:
Change the method `getMovieGenres` to return a Set instead of converting it afterwards in `movie.setGenre()`

Reply:
Changed.